website/sidebars.json

```diff
@@ -143,6 +143,24 @@
               ]
             }
           ]
+        },

```

Comment:
fwiw I don't think this content needs to be on the website - it's not going to be useful for folks _using_ the library.

Reply:
All good, I'll take it off the sidebar and just leave it at docs/internal then!